### PR TITLE
Add the space cadet module

### PR DESCRIPTION
**Pull request by timmow**
Sets up everything in this article if you include modern_space_cadet

http://stevelosh.com/blog/2012/10/a-modern-space-cadet/

### CODE REVIEW

File: Puppetfile [hunk 1]
```diff
@@ -90,6 +90,7 @@ github "karabiner",      "1.0.6"
 github "launchbar",      "1.2.0", :repo => "dieterdemeyer/puppet-launchbar"
 github "libreoffice",    "4.2.1", :repo => "alphagov/puppet-libreoffice"
 github "macvim",         "1.0.0"
+github "modern_space_cadet", "1.0.7"
```
Comment (KushalP) [inline Puppetfile]: Can you align all of the version numbers here please?

Comment (KushalP) [inline Puppetfile]: Not a blocker. Just to stop me from twitching.

**Comment by KushalP** [comment]
space cadet cat

File: Puppetfile (hunk 1 above)
Comment (timmow) [inline Puppetfile]: I thought about that, then noticed `notational_velocity` and `emacs_keybindings` were not aligned, so decided to go with it. Happy to realign if people think that is best though.

Comment (bradleywright) [inline Puppetfile]: Let's do a separate PR for whitespace hygiene.

**Comment by bradleywright** [comment]
Happy keyboarding!
